#ERROR:
try:
  x
except HTTPError as e:
  pass